docs/source/user_guide/config.rst

```diff
@@ -231,6 +238,23 @@ For example, you can customize your FiftyOne config at runtime as follows:
         show_progress_bars=True,
     )
 
+.. _configuring-mongodb-connection:
+
+Configuring a MongoDB connection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, FiftyOne is installed with it's own `mongod` database
+distribution. This database is managed by FiftyOne automatically as a service
+that runs whenever at least one FiftyOne Python client is alive.
+
+As an alternative to this managed database service, a valid
+`MongoDB connection string URI <https://docs.mongodb.com/manual/reference/connection-string/>`_
+can be configured via the `database_uri` setting, and FiftyOne will connect to
+your own MongoDB instance instead. FiftyOne requires MongoDB version `4.4`.
```

Comment:
Are there any concerns with admin DB migrations when the user is using their own database? IE might they fail due to permissions, etc?

If so, should this be documented here?

Reply:
Sure, I'll add a note.